sfdx-source/apex-common-samplecode/main/classes/domains/Opportunities.cls

```diff
@@ -104,13 +104,17 @@ public class Opportunities extends fflib_SObjectDomain
 	{
 		// Related Accounts
 		List<Id> accountIds = new List<Id>();
-		for(Opportunity opp : (List<Opportunity>) Records)
-			if(opp.AccountId!=null)
+		for (Opportunity opp : (List<Opportunity>) Records)
```

Comment:
@daveespo please change "Records" to be lowercase as it is not a class and simply a member variable

Reply:
@ImJohnMDaniel The `Records` start here with a uppercase, because the variable is named in that format. `fflib_SObjectDomain.Records`.
But let me change that into `getRecords()`